lib/tasks/convertFrequencyAndTimeToCron.ts

```
@@ -0,0 +1,47 @@
+    switch (frequency) {
+      case "Daily":
+        return `${minute} ${hour} * * *`;
+      case "Weekly":
+        // Default to Monday (1) for weekly
+        return `${minute} ${hour} * * 1`;
+      case "Monthly":
+        // Default to 1st day of month for monthly
+        return `${minute} ${hour} 1 * *`;
+      case "Once":
+        // For "Once", we'll use the current date
+        const now = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)